src/Bucket/Bucket.jl

```diff
@@ -445,13 +453,9 @@ function make_update_aux(model::BucketModel{FT}) where {FT}
             ),
             surface_space,
         )
-        p.bucket.ρ_sfc .= surface_air_density(
-            model.atmos,
-            p,
-            t,
-            name(model),
-            model.parameters.earth_param_set,
-        )
+        p.bucket.ρ_sfc .=
+            surface_air_density(model.atmos, model, Y, p, t, p.bucket.T_sfc)
```

Comment:
Could we remove the `model.atmos` parameter and instead extract the atmosphere from `model`? Or do we want the interface of `surface_air_density` to take in the atmosphere as a separate parameter?

Reply:
same for `surface_fluxes` and `net_radiation` below

Reply:
All set after discussion